_docs_integrate/error-codes.md

```diff
@@ -24,74 +24,163 @@ required_by:
+| {% include anchor a="error.transport.messages.plaintextMismatch" %}                                               | The own address was not named as a recipient within the signed MessagePlaintext. For example this can be caused by a replay attack.                                                                                                                             |
+| {% include anchor a="error.transport.messages.signatureListMismatch" %}                                           | The signature list didn't contain an entry for a given address.                                                                                                                                                                                                 |
+| {% include anchor a="error.transport.messages.signatureNotValid" %}                                               | The digital signature on a Message for a peer key is invalid. An impersonination attack might be the cause of this.                                                                                                                                             |
+| {% include anchor a="error.transport.notSupported" %}                                                             | The requested method is not yet supported.                                                                                                                                                                                                                      |
+| {% include anchor a="error.transport.recordNotFound" %}                                                           | The requested record was not found.                                                                                                                                                                                                                             |
+| {% include anchor a="error.transport.relationships.activeIdentityDeletionProcessOfOwnerOfRelationshipTemplate" %} | The Identity who created the RelationshipTemplate is currently in the process of deleting itself. Thus, it is not possible to establish a Relationship to it. ([Relationship]({% link _docs_integrate/data-model-overview.md %}#relationship))                  |
+| {% include anchor a="error.transport.relationships.wrongChangeStatus" %}                                          | The Relationship change has the wrong status to run the requested operation. ([Get Relationship]({% link _docs_use-cases/use-case-transport-get-relationship-by-address.md %}), [Relationship]({% link _docs_integrate/data-model-overview.md %}#relationship)) |
```

Comment:
This can also be removed as we removed RelationshipChanges for v5.

Reply:
I think, when we will merge V5 we should do another look over all error codes changes